dtm0/ut/main.c

```diff
@@ -138,6 +138,7 @@ static void dtm0_ut_client_fini(struct cl_ctx *cctx)
 	m0_net_domain_fini(&cctx->cl_ndom);
 }
 
+extern struct m0_semaphore g_test_wait;
```

Comment:
please clean-up this code so that it is not exporting external semaphore used in UTs.